includes/class-apple-news.php

```diff
@@ -365,34 +365,37 @@ public static function has_uninitialized_error(): WP_Error|null {
 
 	/**
 	 * Constructor. Registers action hooks.
-	 *
-	 * @access public
 	 */
 	public function __construct() {
 		add_action(
 			'admin_enqueue_scripts',
 			[ $this, 'action_admin_enqueue_scripts' ]
 		);
+
 		add_action(
 			'enqueue_block_editor_assets',
 			[ $this, 'action_enqueue_block_editor_assets' ]
 		);
+
 		add_action(
-			'plugins_loaded',
-			[ $this, 'action_plugins_loaded' ]
+			'init',
```

Comment:
I learned as part of this fix that loading `__(` translations before init has always been a mistake. The difference now is that WordPress is making that obvious to developers like me and encouraging us to hook translable code in the proper hook.